src/loader.css

```diff
@@ -13,6 +13,8 @@
   align-items: center;
   flex-direction: column;
   position: relative;
+  z-index: 999999;
```

Comment:
hey @ihordubas1 I added this line in css to keep loader in foreground. Is there a more elegant way to do this?

Reply:
It's ok to write like this in some special cases, so we can keep it

Reply:
thanks! :)